docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md

```diff
@@ -7,7 +7,8 @@ title: "Adding & Removing Recipes"
 
 ## Removing Recipes
 
-Removing GTCEu Modern recipes with KubeJS works the same as any other recipe, meaning they can be removed by: ID, Mod, Input, Output, Type or a Mixture.
+Removing GTCEu Modern recipes with KubeJS works the same as any other recipe, meaning they can be removed by: 
+ID, Mod, Input, Output, Type or a Mixture.
```

Comment:
Probably decapitalise Mixture, it seems to imply that Mixture itself is a parameter instead of being a mixture of removal parameters